web-app/pages/certificate/[tokenId].tsx

```diff
@@ -38,16 +39,31 @@ function buildLinkedInUrl(pngUrl: string) {
 //   return { props };
 // };
 
-const Certificate: NextPage = () => {
+// eslint-disable-next-line max-lines-per-function
+const Certificate: NextPage = (...props) => {
   const router = useRouter();
-  const { tokenId } = router.query; // https://nextjs.org/docs/routing/dynamic-routes
-
-  console.log({ tokenId });
-  const pngUrl = `${baseUrl}/api/cert/${tokenId}.png`;
-
+  const { tokenId } = router.query; // https://nextjs.org/docs/routing/dynamic-routes  
+  const data = tokenId;
+  const pngUrl = `${baseUrl}/api/cert/${data}.png`;
+  // eslint-disable-next-line react/destructuring-assignment
+  console.log('************************', { data });
   return (
     <Layout>
-      <a href={`/api/cert/${tokenId}.svg`} className="md:max-w-xl lg:max-w-2xl">
+      <Head>
+        <meta property="og:url" content={`/api/cert/${data}`} />
+        <meta property="og:type" content="website" />
+        <meta property="og:title" content="I got certified with NEAR university" />
+        <meta property="og:description" content="View NEAR University certificates of any .near account" />
+        <meta property="og:image" content={`/api/cert/${data}.png`} />
+        <meta property="twitter:site" content="@NEARProtocol" />
+        <meta name="twitter:card" content="summary_large_image" />
+        <meta property="twitter:domain" content="/" />
+        <meta property="twitter:url" content={`/certificate/${data}`} />
+        <meta name="twitter:title" content="I got certified with NEAR university" />
+        <meta name="twitter:description" content="View NEAR University certificates of any .near account" />
+        <meta name="twitter:image" content={`/api/cert/${data}.png`} />
```

Comment:
@rashaabdulrazzak  Probably most of the URLs here need to be absolute instead of relative.  Also, don't recreate them as strings each time. Just use a const that we defined above, such as when I had `const pngUrl = `${baseUrl}/api/cert/${tokenId}.png`;`.

Reply:
I will replace them